tests/queries/0_stateless/01034_JSONCompactEachRow.sql

```diff
@@ -22,51 +22,64 @@ SELECT 5;
 /* Check JSONCompactEachRow Input */
 CREATE TABLE test_table (v1 String, v2 UInt8, v3 DEFAULT v2 * 16, v4 UInt8 DEFAULT 8) ENGINE = MergeTree() ORDER BY v2;
 INSERT INTO test_table FORMAT JSONCompactEachRow ["first", 1, "2", null] ["second", 2, null, 6];
+
```

Comment:
according to Alexey Milovidov's proposal, i use "\n\n" as the delimiter for insert into statements(exclude format=values). This approach results in fewer parsing issues because the variety of format issues can lead to more problems that are not easily resolved with the use of ";\n" or "\n".(For example, using "\n" makes it difficult to handle multiline statements; using ";\n" is not compatible with TSV and CSV formats, and it's challenging to determine whether ';' is part of the data or a delimiter.）

Reply:
We may make the new mode an option, for example, single_query_multi_line. When it's enabled, use this pr mode(split by semicolon or empty line, a single user query can have multiple lines);  if it's not enabled, follow the original mode(split by \n). But overall, multiquery is supported by default.

Reply:
I think it's not worth it... Let's change the behavior without an option.